test/test_cookbook_geometry.jl

```diff
@@ -1,770 +1,981 @@
+            (1214704.933941905, 641092.8288590391),
+            (1228580.428455506, 682719.3123998424),
+            (1218405.0658121984, 721108.1805541387),
+            (1179091.1646903288, 712782.8838459781),
+        ]) do poly
+            @test AG.toJSON(poly)[1:19] == "{ \"type\": \"Polygon\""
+            @test AG.toGML(poly)[1:13] == "<gml:Polygon>"
+            @test AG.toWKT(poly)[1:7] == "POLYGON"
+            @test AG.toWKB(poly) == UInt8[
+                0x01,
+                0x03,
+                0x00,
+                0x00,
+                0x00,
+                0x01,
+                0x00,
+                0x00,
+                0x00,
+                0x06,
+                0x00,
+                0x00,
+                0x00,
+                0x38,
+                0x25,
+                0x29,
+                0x2a,
+                0xd3,
+                0xfd,
+                0x31,
+                0x41,
+                0xc5,
+                0x75,
+                0x87,
+                0xc4,
+                0x9d,
+                0xc0,
+                0x25,
+                0x41,
+                0x45,
+                0x2b,
+                0x96,
+                0x05,
+                0x5d,
+                0xb7,
+                0x31,
+                0x41,
+                0xf8,
+                0x4b,
+                0x70,
+                0x89,
+                0x80,
+                0x5e,
+                0x24,
+                0x41,
+                0x12,
+                0xd1,
+                0x16,
+                0xef,
+                0xf0,
+                0x88,
+                0x32,
+                0x41,
+                0x44,
+                0x36,
+                0x60,
+                0xa8,
+                0x89,
+                0x90,
+                0x23,
+                0x41,
+                0x92,
+                0x42,
+                0xaf,
+                0x6d,
+                0x24,
+                0xbf,
+                0x32,
+                0x41,
+                0x45,
+                0xdf,
+                0xf2,
+                0x9f,
+                0xbe,
+                0xd5,
+                0x24,
+                0x41,
+                0x78,
+                0x11,
+                0xd9,
+                0x10,
+                0x65,
+                0x97,
+                0x32,
+                0x41,
+                0x92,
+                0x97,
+                0x71,
+                0x5c,
+                0xa8,
+                0x01,
+                0x26,
+                0x41,
+                0x38,
+                0x25,
+                0x29,
+                0x2a,
+                0xd3,
+                0xfd,
+                0x31,
+                0x41,
+                0xc5,
+                0x75,
+                0x87,
+                0xc4,
+                0x9d,
+                0xc0,
+                0x25,
+                0x41,
+            ]
```

Comment:
This could be worth a https://domluna.github.io/JuliaFormatter.jl/dev/skipping_formatting/

```
#! format: off
<this test>
#! format: on
```

Reply:
Reflected in https://github.com/yeesian/ArchGDAL.jl/pull/179/commits/e86beac5117657411ed42ff1f269003db0bd6f69, thanks!